### ISSUE TITLE
Treating RFC organization authors as organizations

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-ietf#102]
As per https://github.com/ietf-tools/bibxml-service/issues/296#issuecomment-1273389794 request from the RPC, we are to distinguish the three classes of authors.

We will revert the change that introduced this:
- https://github.com/ietf-tools/relaton-data-rfcs/blob/742e255da9bdcec26907124a9660be2109fa8758/data/RFC3716.yaml#L25-L28

```yaml
  role:
  - description:
    - content: BibXML author
    type: author
```

## Type 1: Personal authors

No affiliation:
```xml
<author fullname="..." initials="..." surname="..."/>
```

```yaml
contributor:
- person:
    name:
      given:
        forename:
          initial: ...
          initial: ...
        formatted_initials:
          content: ...
      surname:
        content: ...
      completename:
        content: ...
```

With affiliation:
```xml
<author fullname="..." initials="..." surname="...">
  <organization>...</organization>
</author>
```

```yaml
contributor:
- person:
    name:
      given:
        forename:
          initial: ...
          initial: ...
        formatted_initials:
          content: ...
      surname:
        content: ...
      completename:
        content: ...
    affiliation: ...
```

## Type 2: Organizational authors

```xml
<author>
  <organization>...</organization>
</author>
```

```yaml
contributor:
- organization:
    name:
    - content: ...
```

## Type 3: Undetermined type authors

```xml
<author fullname="..."/>
```

```yaml
contributor:
- person:
    name:
      completename:
        content: ...
```

## List to categorize

The outcome of https://github.com/ietf-tools/bibxml-service/issues/296#issuecomment-1273456660 will determine the categorization of this list of authors which is filtered from the full rfc-index.xml (filtered away all personal names).

```
ACM SIGUCCS
Audio-Video Transport Working Group
Bolt Beranek
Defense Advanced Research Projects Agency
EARN Staff
ESCC X.500/X.400 Task Force
ESnet Site Coordinating Comittee (ESCC)
End-to-End Services Task Force
Energy Sciences Network (ESnet)
Federal Networking Council
Gateway Algorithms and Data Structures Task Force
IAB Advisory Committee
IAB and IESG
IAB
IANA
IESG
IETF Secretariat
ISO
ISOC Board of Trustees
Information Sciences Institute University of Southern California
International Organization for Standardization
International Telegraph and Telephone Consultative Committee of the International Telecommunication Union
Internet Activities Board
Internet Architecture Board
Internet Assigned Numbers Authority (IANA)
Internet Engineering Steering Group
KOI8-U Working Group
Mitra
National Bureau of Standards
National Research Council
National Science Foundation
NetBIOS Working Group in the Defense Advanced Research Projects Agency
Network Information Center. Stanford Research Institute
Network Technical Advisory Group
Newman Laboratories
North American Directory Forum
Sun Microsystems
The Internet Society
The North American Directory Forum
Vietnamese Standardization Working Group
```

**Comment by ronaldtse** [comment]
This task is ready to be done.

**Comment by andrew2net** [comment]
@ronaldtse some items of the "List to categorize" are missing in the [test cases](https://github.com/relaton/relaton-ietf/issues/103#issue-1407026136). Should we add test cases for them?

**Comment by ronaldtse** [comment]
@andrew2net yes please, that would be helpful. Thanks!

**Comment by ronaldtse** [comment]
@andrew2net as per https://github.com/ietf-tools/bibxml-service/issues/296#issuecomment-1284138158 by @ajeanmahoney, the "RFC Editor, et al." author should also be handled as an organization:

```
<rfc-entry>
<doc-id>RFC2555</doc-id>
<title>30 Years of RFCs</title>
<author>
<name>RFC Editor, et al.</name>
</author>
```

and the corresponding BibXML becomes:

```xml
<reference anchor="RFC2555" target="https://www.rfc-editor.org/info/rfc2555">
<front>
<title>30 Years of RFCs</title>
<author>
<organization>RFC Editor, et al.</organization>
</author>
```

Thanks!

**Comment by andrew2net** [comment]
Fixed in v 1.13.5. Here are the BibXML test outputs https://github.com/relaton/relaton-bib/blob/main/spec/examples/rfc.xml

**Comment by ronaldtse** [comment]
Thanks @andrew2net, I can see it is addressed here: https://github.com/relaton/relaton-ietf/commit/f4865fa7c0be53253aa3aabc808ea65afaaf7f05
